### YamlMime:ManagedReference
items:
- uid: System.Security.Cryptography.CryptographicException
  id: CryptographicException
  children:
  - System.Security.Cryptography.CryptographicException.#ctor
  - System.Security.Cryptography.CryptographicException.#ctor(System.Int32)
  - System.Security.Cryptography.CryptographicException.#ctor(System.String)
  - System.Security.Cryptography.CryptographicException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Security.Cryptography.CryptographicException.#ctor(System.String,System.Exception)
  - System.Security.Cryptography.CryptographicException.#ctor(System.String,System.String)
  langs:
  - csharp
  name: CryptographicException
  nameWithType: CryptographicException
  fullName: System.Security.Cryptography.CryptographicException
  type: Class
  summary: "Unntaket oppstod når det oppstår en feil under en kryptografisk operasjon."
  remarks: "CryptographicException bruker HRESULT CORSEC_E_CRYPTO, med mindre en alternativ feilkode er angitt.       Hvis du vil se en liste med opprinnelige verdier for en forekomst av CryptographicException, kan du se konstruktøren."
  example:
  - "The following code example demonstrates how to use members of the CryptographicException class.  \n  \n [!code-cs[System.Security.Cryptography.CryptographicException#2](~/add/codesnippet/csharp/t-system.security.crypto_115_1.cs)]\n [!code-vb[System.Security.Cryptography.CryptographicException#2](~/add/codesnippet/visualbasic/t-system.security.crypto_115_1.vb)]\n [!code-cpp[System.Security.Cryptography.CryptographicException#2](~/add/codesnippet/cpp/t-system.security.crypto_115_1.cpp)]"
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public class CryptographicException : SystemException
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Security.Cryptography.CryptographicException.#ctor
  id: '#ctor'
  parent: System.Security.Cryptography.CryptographicException
  langs:
  - csharp
  name: CryptographicException()
  nameWithType: CryptographicException.CryptographicException()
  fullName: System.Security.Cryptography.CryptographicException.CryptographicException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security.Cryptography
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Security.Cryptography.CryptographicException&quot;> </xref> klasse standardegenskapene."
  remarks: "Tabellen nedenfor viser første egenskapsverdiene for en forekomst av <xref:System.Security.Cryptography.CryptographicException>.</xref:System.Security.Cryptography.CryptographicException>      | Egenskapen | Verdien |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Lokaliserte feil melding strengen. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  example:
  - "The following code example demonstrates how to construct a <xref:System.Security.Cryptography.CryptographicException> with no parameters. This code example is part of a larger example provided for the <xref:System.Security.Cryptography.CryptographicException> class.  \n  \n [!code-cs[System.Security.Cryptography.CryptographicException#1](~/add/codesnippet/csharp/m-system.security.crypto_47_1.cs)]\n [!code-vb[System.Security.Cryptography.CryptographicException#1](~/add/codesnippet/visualbasic/m-system.security.crypto_47_1.vb)]\n [!code-cpp[System.Security.Cryptography.CryptographicException#1](~/add/codesnippet/cpp/m-system.security.crypto_47_1.cpp)]"
  syntax:
    content: public CryptographicException ();
    parameters: []
  overload: System.Security.Cryptography.CryptographicException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.Cryptography.CryptographicException.#ctor(System.Int32)
  id: '#ctor(System.Int32)'
  parent: System.Security.Cryptography.CryptographicException
  langs:
  - csharp
  name: CryptographicException(Int32)
  nameWithType: CryptographicException.CryptographicException(Int32)
  fullName: System.Security.Cryptography.CryptographicException.CryptographicException(Int32)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security.Cryptography
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Security.Cryptography.CryptographicException&quot;> </xref> klassen med den angitte <xref uid=&quot;langword_csharp_HRESULT&quot; name=&quot;HRESULT&quot; href=&quot;&quot;> </xref> feilkoden."
  remarks: "CryptographicException-konstruktøren aksepterer bare et `HRESULT` feilkoden og angir de <xref:System.Exception.Message%2A>egenskapen til en lokalisert melding som tilsvarer det `HRESULT`.</xref:System.Exception.Message%2A>       Tabellen nedenfor viser første egenskapsverdiene for en forekomst av <xref:System.Security.Cryptography.CryptographicException>.</xref:System.Security.Cryptography.CryptographicException>      | Egenskapen | Verdien |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Lokaliserte feil melding strengen. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  example:
  - "The following code example demonstrates how to construct a <xref:System.Security.Cryptography.CryptographicException> using an error code for an <xref:System.Security.Cryptography.CryptographicUnexpectedOperationException>. This code example is part of a larger example provided for the <xref:System.Security.Cryptography.CryptographicException> class.  \n  \n [!code-cs[System.Security.Cryptography.CryptographicException#3](~/add/codesnippet/csharp/m-system.security.crypto_36_1.cs)]\n [!code-vb[System.Security.Cryptography.CryptographicException#3](~/add/codesnippet/visualbasic/m-system.security.crypto_36_1.vb)]\n [!code-cpp[System.Security.Cryptography.CryptographicException#3](~/add/codesnippet/cpp/m-system.security.crypto_36_1.cpp)]"
  syntax:
    content: public CryptographicException (int hr);
    parameters:
    - id: hr
      type: System.Int32
      description: "Den <xref uid=&quot;langword_csharp_HRESULT&quot; name=&quot;HRESULT&quot; href=&quot;&quot;> </xref> feilkoden."
  overload: System.Security.Cryptography.CryptographicException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.Cryptography.CryptographicException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Security.Cryptography.CryptographicException
  langs:
  - csharp
  name: CryptographicException(String)
  nameWithType: CryptographicException.CryptographicException(String)
  fullName: System.Security.Cryptography.CryptographicException.CryptographicException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security.Cryptography
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Security.Cryptography.CryptographicException&quot;> </xref> klassen med en angitt feilmelding."
  remarks: "Tabellen nedenfor viser første egenskapsverdiene for en forekomst av <xref:System.Security.Cryptography.CryptographicException>.</xref:System.Security.Cryptography.CryptographicException>      | Egenskapen | Verdien |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Feil melding strengen. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  example:
  - "The following code example demonstrates how to construct a <xref:System.Security.Cryptography.CryptographicException> using a custom error message. This code example is part of a larger example provided for the <xref:System.Security.Cryptography.CryptographicException> class.  \n  \n [!code-cs[System.Security.Cryptography.CryptographicException#4](~/add/codesnippet/csharp/m-system.security.crypto_116_1.cs)]\n [!code-vb[System.Security.Cryptography.CryptographicException#4](~/add/codesnippet/visualbasic/m-system.security.crypto_116_1.vb)]\n [!code-cpp[System.Security.Cryptography.CryptographicException#4](~/add/codesnippet/cpp/m-system.security.crypto_116_1.cpp)]"
  syntax:
    content: public CryptographicException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Feilmelding som forklarer hvorfor unntaket."
  overload: System.Security.Cryptography.CryptographicException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.Cryptography.CryptographicException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Security.Cryptography.CryptographicException
  langs:
  - csharp
  name: CryptographicException(SerializationInfo,StreamingContext)
  nameWithType: CryptographicException.CryptographicException(SerializationInfo,StreamingContext)
  fullName: System.Security.Cryptography.CryptographicException.CryptographicException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security.Cryptography
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Security.Cryptography.CryptographicException&quot;> </xref> klasse serialiserte dataene."
  remarks: "Denne konstruktøren kalles under deserialisering til reconstitute unntak objektet overføres via en dataflyt."
  syntax:
    content: protected CryptographicException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "Objektet som har datatypen serialiserte objektet."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "Kontekstavhengig informasjon om kilden eller målet."
  overload: System.Security.Cryptography.CryptographicException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.Cryptography.CryptographicException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Security.Cryptography.CryptographicException
  langs:
  - csharp
  name: CryptographicException(String,Exception)
  nameWithType: CryptographicException.CryptographicException(String,Exception)
  fullName: System.Security.Cryptography.CryptographicException.CryptographicException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security.Cryptography
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Security.Cryptography.CryptographicException&quot;> </xref> klassen med en angitt feilmelding og en referanse til det interne unntaket er årsaken til dette unntaket."
  remarks: "Et unntak som er registrert som en direkte resultatet av et tidligere unntak må inkludere en referanse til det tidligere unntaket i den <xref:System.Exception.InnerException%2A>egenskapen.</xref:System.Exception.InnerException%2A> Den <xref:System.Exception.InnerException%2A>egenskapen returnerer den samme verdien som er sendt til konstruktøren, eller `null` hvis den <xref:System.Exception.InnerException%2A>egenskapen leverer ikke internt unntak verdien til konstruktøren.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       Tabellen nedenfor viser første egenskapsverdiene for en forekomst av <xref:System.Security.Cryptography.CryptographicException>.</xref:System.Security.Cryptography.CryptographicException>      | Egenskapen | Verdien |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Referansen for det indre unntaket. |   | <xref:System.Exception.Message%2A>| Feil melding strengen. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  example:
  - "The following code example demonstrates how to construct a <xref:System.Security.Cryptography.CryptographicException> using a custom error message and an inner exception. This code example is part of a larger example provided for the <xref:System.Security.Cryptography.CryptographicException> class.  \n  \n [!code-cs[System.Security.Cryptography.CryptographicException#5](~/add/codesnippet/csharp/d23f33f8-33a5-45ad-932a-_1.cs)]\n [!code-vb[System.Security.Cryptography.CryptographicException#5](~/add/codesnippet/visualbasic/d23f33f8-33a5-45ad-932a-_1.vb)]\n [!code-cpp[System.Security.Cryptography.CryptographicException#5](~/add/codesnippet/cpp/d23f33f8-33a5-45ad-932a-_1.cpp)]"
  syntax:
    content: public CryptographicException (string message, Exception inner);
    parameters:
    - id: message
      type: System.String
      description: "Feilmelding som forklarer hvorfor unntaket."
    - id: inner
      type: System.Exception
      description: "Unntaket er årsaken til den gjeldende unntaket. Hvis den `inner` -parameteren er ikke <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, oppstår det gjeldende unntaket i en <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> blokken som behandler det interne unntaket."
  overload: System.Security.Cryptography.CryptographicException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.Cryptography.CryptographicException.#ctor(System.String,System.String)
  id: '#ctor(System.String,System.String)'
  parent: System.Security.Cryptography.CryptographicException
  langs:
  - csharp
  name: CryptographicException(String,String)
  nameWithType: CryptographicException.CryptographicException(String,String)
  fullName: System.Security.Cryptography.CryptographicException.CryptographicException(String,String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security.Cryptography
  summary: "Initialiserer en ny forekomst av den <xref href=&quot;System.Security.Cryptography.CryptographicException&quot;> </xref> klassen med feilmeldingen angitt i det angitte formatet."
  remarks: "Tabellen nedenfor viser første egenskapsverdiene for en forekomst av <xref:System.Security.Cryptography.CryptographicException>.</xref:System.Security.Cryptography.CryptographicException>      | Egenskapen | Verdien |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| Feil melding strengen. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  example:
  - "The following code example demonstrates how to construct a <xref:System.Security.Cryptography.CryptographicException> using a time format and the current date. This code example is part of a larger example provided for the <xref:System.Security.Cryptography.CryptographicException> class.  \n  \n [!code-cs[System.Security.Cryptography.CryptographicException#6](~/add/codesnippet/csharp/b79fae48-8bdf-49eb-8d86-_1.cs)]\n [!code-vb[System.Security.Cryptography.CryptographicException#6](~/add/codesnippet/visualbasic/b79fae48-8bdf-49eb-8d86-_1.vb)]\n [!code-cpp[System.Security.Cryptography.CryptographicException#6](~/add/codesnippet/cpp/b79fae48-8bdf-49eb-8d86-_1.cpp)]"
  syntax:
    content: public CryptographicException (string format, string insert);
    parameters:
    - id: format
      type: System.String
      description: "Formatet som brukes til å sende feilmeldingen."
    - id: insert
      type: System.String
      description: "Feilmelding som forklarer hvorfor unntaket."
  overload: System.Security.Cryptography.CryptographicException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.Security.Cryptography.CryptographicException.#ctor
  parent: System.Security.Cryptography.CryptographicException
  isExternal: false
  name: CryptographicException()
  nameWithType: CryptographicException.CryptographicException()
  fullName: System.Security.Cryptography.CryptographicException.CryptographicException()
- uid: System.Security.Cryptography.CryptographicException.#ctor(System.Int32)
  parent: System.Security.Cryptography.CryptographicException
  isExternal: false
  name: CryptographicException(Int32)
  nameWithType: CryptographicException.CryptographicException(Int32)
  fullName: System.Security.Cryptography.CryptographicException.CryptographicException(Int32)
- uid: System.Int32
  parent: System
  isExternal: true
  name: Int32
  nameWithType: Int32
  fullName: System.Int32
- uid: System.Security.Cryptography.CryptographicException.#ctor(System.String)
  parent: System.Security.Cryptography.CryptographicException
  isExternal: false
  name: CryptographicException(String)
  nameWithType: CryptographicException.CryptographicException(String)
  fullName: System.Security.Cryptography.CryptographicException.CryptographicException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Security.Cryptography.CryptographicException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Security.Cryptography.CryptographicException
  isExternal: false
  name: CryptographicException(SerializationInfo,StreamingContext)
  nameWithType: CryptographicException.CryptographicException(SerializationInfo,StreamingContext)
  fullName: System.Security.Cryptography.CryptographicException.CryptographicException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Security.Cryptography.CryptographicException.#ctor(System.String,System.Exception)
  parent: System.Security.Cryptography.CryptographicException
  isExternal: false
  name: CryptographicException(String,Exception)
  nameWithType: CryptographicException.CryptographicException(String,Exception)
  fullName: System.Security.Cryptography.CryptographicException.CryptographicException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Security.Cryptography.CryptographicException.#ctor(System.String,System.String)
  parent: System.Security.Cryptography.CryptographicException
  isExternal: false
  name: CryptographicException(String,String)
  nameWithType: CryptographicException.CryptographicException(String,String)
  fullName: System.Security.Cryptography.CryptographicException.CryptographicException(String,String)
- uid: System.Security.Cryptography.CryptographicException.#ctor*
  parent: System.Security.Cryptography.CryptographicException
  isExternal: false
  name: CryptographicException
  nameWithType: CryptographicException.CryptographicException
